monkey/infection_monkey/utils/capture_output.py

```diff
@@ -0,0 +1,22 @@
+import io
+import sys
+
+
+class StdoutOutputCaptor:
```

Comment:
It would be more pythonic to rename class to `StdoutCapture` and use it like:
```
with StdoutCapture() as capture_string:
    do_something()
    string_to_parse = capture_string
```